api/istio/istio.go

```diff
@@ -84,8 +84,9 @@ func (c *client) PatchVirtualService(ctx context.Context, namespace string, vs *
 	if err != nil {
 		return nil, err
 	}
-
-	return c.networking.VirtualServices(namespace).Patch(vs.ObjectMeta.Name, types.MergePatchType, vsJSON)
+	xx, yy := c.networking.VirtualServices(namespace).Patch(vs.ObjectMeta.Name, types.MergePatchType, vsJSON)
```

Comment:
Maybe a better variable naming?

Reply:
it is not supposed to there, forgot to remove